xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -352,19 +355,23 @@
     #if ("$!dateFormat" == '')
       #set ($dateFormat = $xwiki.getXWikiPreference('dateformat', 'yyyy/MM/dd HH:mm'))
     #end
-    <input id="xwiki-livetable-${htmlLiveTableId}-filter-${foreach.count}" type="text"
-      data-type="date" data-dateformat="$escapetool.xml($dateFormat)"
-      #if ("$!columnProperties.size" != '')size="$!escapetool.xml($columnProperties.size)"#end
-      title="$escapetool.xml($services.localization.render('platform.livetable.filtersTitle',
-      [$services.localization.render("${transprefix}$column")]))" />
+    <div role="combobox" id="xwiki-livetable-${htmlLiveTableId}-filter-date-combobox-${foreach.count}"
+      class="xwiki-livetable-display-header-filter-date-combobox" aria-expanded="false">
+      <input id="xwiki-livetable-${htmlLiveTableId}-filter-${foreach.count}" type="text"
+        data-type="date" data-dateformat="$escapetool.xml($dateFormat)"
+        #if ("$!columnProperties.size" != '')size="$!escapetool.xml($columnProperties.size)"#end
+        title="$!selectTitle"
+        aria-description="$!escapetool.xml($services.localization.render('platform.livetable.filter.date.format.description',[$escapetool.xml($dateFormat)]))"/>
```

Comment:
The inner escaping around `$dateFormat` should be removed, or the date format will be escaped twice.